src/test/resources/metafiles/valid/from-spec.json

```diff
@@ -15,6 +15,7 @@
       "next_action": "return",
       "next_action_date": "24-02-2017 00:00:00.000010",
       "file_name": "1111001.pdf",
+      "ocr_data": "VGVzdCBvY3IgZGF0YQo=",
```

Comment:
do we know whether this will be a base64 encoded json?

Reply:
yes, we [do](https://tools.hmcts.net/confluence/display/RBS/Technical+Specification+-+Exela#TechnicalSpecification-Exela-3.4Metafilefields)